packages/client/src/client.ts

```diff
@@ -80,19 +73,30 @@ export class DefaultDelegatedRoutingV1HttpApiClient implements DelegatedRoutingV
     return this.started
   }
 
-  start (): void {
+  async start (): Promise<void> {
```

Comment:
I noticed that `start` isn't defined on the `DelegatedRoutingV1HttpApiClient` type. 

Should we add it?

Reply:
Makes sense.  I've also made the stop method async on the interface because it's async now..